meta/Meta.cpp

```diff
@@ -6954,7 +6976,8 @@ void Meta::meta_sai_on_fdb_flush_event_consolidated(
             continue;
         }
 
-        if (fdbTypeAttr->getSaiAttr()->value.s32 != type->value.s32)
+        if ((type->value.s32 != SAI_FDB_FLUSH_ENTRY_TYPE_ALL) &&
```

Comment:
How about this?
```
if (type->value.s32 == SAI_FDB_FLUSH_ENTRY_TYPE_ALL) {
    // add fdb to remove vector and continue;
}

// check type match
// check bridge_port_id matches
// check vlan ID matches
```

If you dont like this, then when someone specified type == SAI_FDB_FLUSH_ENTRY_TYPE_ALL, and also provides vlan_id or bridge_port_id or both, we are NOT going to flush all. Is that the expected behavior? Or we should honor the type attribute?

Reply:
Also, what do we print on line number 6963 when value is SAI_FDB_FLUSH_ENTRY_TYPE_ALL? Because we can casting from FLUSH_ENTRY_TYPE to FDB_ENTRY_TYPE enum

Reply:
cant be like that , since if it's ALL (static and dynamic, other attributes also needs to be checked, like vlan id, or bridge port, ALL here appllies only to static or dynamic

Reply:
Got it. SAI_FDB_FLUSH_ENTRY_TYPE_ALL means BOTH(STATIC and DYNAMIC) type! NOT really ALL FDB entries!

Reply:
there are no other entries except static and dynamic, so ALL means static and dynamic together